### PR TITLE
Replaced strong etags with HTTP ETags

### LINKED ISSUE #1730: Edit HTTP remote related posts regarding strong and weak etag
After https://github.com/iterative/dvc/pull/4372 we started supporting weak etags.
Need to update the documents related.

- [x] https://dvc.org/doc/user-guide/dvc-files-and-directories
- [x] https://dvc.org/doc/user-guide/external-dependencies
- [x] https://dvc.org/doc/command-reference/import-url

### PR DESCRIPTION
**Pull request by sarthakforwet**
Fixes #1730.

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Hi @sarthakforwet does this PR intend to close #1730? If so please link them. See https://github.blog/2013-05-14-closing-issues-via-pull-requests/

File: content/docs/command-reference/import-url.md [hunk 1]
```diff
@@ -55,18 +55,18 @@ source.
...
-| `remote` | Remote path\*                     | `remote://remote-name/data`                |
+| Type     | Description                     | `url` format example                       |
+| -------- | ------------------------------- | ------------------------------------------ |
+| `s3`     | Amazon S3                       | `s3://bucket/data`                         |
+| `azure`  | Microsoft Azure Blob Storage    | `azure://container/data`                   |
+| `gdrive` | Google Drive                    | `gdrive://<folder-id>/data`                |
+| `gs`     | Google Cloud Storage            | `gs://bucket/data`                         |
+| `ssh`    | SSH server                      | `ssh://user@example.com/path/to/data`      |
+| `hdfs`   | HDFS to file\*                  | `hdfs://user@example.com/path/to/data.csv` |
+| `http`   | HTTP to file with _HTTP ETag_\* | `https://example.com/path/to/data.csv`     |
```
Comment (jorgeorpinel) [inline content/docs/command-reference/import-url.md:66]: No need to repeat "HTTP" here:
```suggestion
| `http`   | HTTP to file with _ETag_\* | `https://example.com/path/to/data.csv`     |
```
> Needs reformatting though.

File: content/docs/command-reference/import-url.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ DVC supports several types of (local or) remote locations (protocols):
   single files.
 
 - In case of HTTP,
-  [strong ETag](https://en.wikipedia.org/wiki/HTTP_ETag#Strong_and_weak_validation)
+  [HTTP ETag](https://en.wikipedia.org/wiki/HTTP_ETag#Strong_and_weak_validation)
```
Comment (jorgeorpinel) [inline content/docs/command-reference/import-url.md:83]: Same: We have "...of HTTP, HTTP eTag" with this change.

File: content/docs/user-guide/dvc-files-and-directories.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ An _output entry_ (`outs`) consists of these fields:
   the file's location)
 - `md5`, `etag`, or `checksum`: Hash value for the file or directory being
   tracked with DVC. MD5 is used for most locations (local file system and SSH);
-  [strong ETag](https://en.wikipedia.org/wiki/HTTP_ETag#Strong_and_weak_validation)
+  [HTTP ETag](https://en.wikipedia.org/wiki/HTTP_ETag#Strong_and_weak_validation)
   for HTTP, S3, or Azure
```
Comment (jorgeorpinel) [inline content/docs/user-guide/dvc-files-and-directories.md:71]: Same. This reads "HTTP ETag for HTTP".

**Review by jorgeorpinel — CHANGES_REQUESTED** [review]
Thanks @sarthakforwet but so far this looks like a grep-replace. It's a start but it's necessary to read the context around the change to see it makes sense please. Left a few specific instances but this is probably pervasive in all the changes. Please lmk when to review again.

**Comment by sarthakforwet** [comment]
> but this is probably pervasive in all the changes.

I also had the thought of changing the phrase according to surroundings but thought that it was necessary to add complete "HTTP ETag". Corrected now.

**Review by shcheklein — APPROVED** [review]

**Review by jorgeorpinel — APPROVED** [review]
Looks good now, thanks @sarthakforwet!

**Pull request merged**